src/openrct2/world/Climate.cpp

```diff
@@ -59,6 +59,9 @@ ClimateState gClimateCurrent;
 uint16_t gClimateUpdateTimer;
 uint16_t gClimateLightningFlash;
 
+// GameState
+static GameState_t gameState;
+
```

Comment:
Allocating a new gamestate? This doesn't seem right.

Reply:
Would I need to use instead?
```cpp
auto& gameState = GetGameState();
```

Reply:
You'd want a const reference, yes. Preferably not a new global, but at function or class level. At the moment, I'm not even sure it's assigned or initialised anywhere, though?